tests/Aspire.Hosting.Containers.Tests/ContainerResourceBuilderTests.cs

```diff
@@ -54,8 +54,8 @@ public void WithImageMutatesImageNameOfLastAnnotation()
     public void WithImageTagMutatesImageTag()
     {
         using var builder = TestDistributedApplicationBuilder.Create();
-        var redis = builder.AddContainer("redis", "redis").WithImageTag("foo");
-        Assert.Equal("foo", redis.Resource.Annotations.OfType<ContainerImageAnnotation>().Single().Tag);
+        var redis = builder.AddRedis("redis").WithImageTag("7.1");
+        Assert.Equal("7.1", redis.Resource.Annotations.OfType<ContainerImageAnnotation>().Single().Tag);
```

Comment:
Should this be reverted? Why it is using Redis in the core "Hosting.Containers" tests?

Reply:
Reverted